test/python/utest_masarclient/test_gatherV3Data.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from types import *
+
+from masarclient.gatherV3Data import GatherV3Data as GatherV3Data
+from masarclient.ntmultiChannel import NTMultiChannel as NTMultiChannel
+from masarclient.alarm import Alarm as Alarm
+from masarclient.timeStamp import TimeStamp as TimeStamp
+
+
+'''
+
+Unittests for masarService/python/masarclient/alarm.py
+
+'''
+
+
+class TestGatherV3Data(unittest.TestCase):
+
+    '''
+    These names are meant to be used with the test database in its present state and may need to be changed.
+    '''
+    def setUp(self):
+        self.connected = False
+        self.names = (
+            'masarExampleBoUninit',
+            'masarExampleMbboUninit',
+            'masarExample0002',
+            'masarExample0003',
+            'masarExample0000',
+            'masarExample0001',
+            'masarExample0004',
+            'masarExampleCharArray',
+            'masarExampleStringArray',
+            'masarExampleShortArray',
+            'masarExampleLongArray',
+            'masarExampleFloatArray',
+            'masarExampleDoubleArray',
+        )
+        self.gatherv3data = GatherV3Data(self.names)
+        self.connected = self.gatherv3data.connect(2.0)
```

Comment:
setUp() can have assert*() calls in it.  So ```self.assertTrue(self.connected)``` can be here instead of in every method.